### PR TITLE
improve readability on stateless contract beginner examples, fix to b…

### PR DESCRIPTION
**Pull request by edwardgaudio**
**Is your addition related to a problem? Please describe.**
In many places in the docs `let` or `var` are used instead of `const`. In this particular area of the code, I've changed `var` and `let` to `const` where applicable. I also re-organized imports to be consistent with node/js best practices. Lastly I went with a `main` function running with a promise at the bottom. I saw this in other docs and I thought it looked very clear and clean.

**Describe alternatives you've considered**
I considered with the account delegation signing to just show the difference in code from the code example above or putting the full example in there again. In the end I went with just the main function with a note in the comments and all other non-relevant and redundant comments removed.

### CODE REVIEW

File: docs/features/asc1/stateless/sdks.md [hunk 1]
```diff
@@ -257,12 +262,18 @@ The SDKs require that parameters to a stateless smart contract TEAL program be i
...
 ```javascript tab="JavaScript"
-    //string parameter
-    let args = ["my string"];
-    let lsig = algosdk.makeLogicSig(program, args);
-    //integer parameter
-    let args = [[123]];
-    let lsig = algosdk.makeLogicSig(program, args);
+    // string parameter
+    const args = ["my string"];
+    const lsig = algosdk.makeLogicSig(program, args);
```
Comment (edwardgaudio) [inline docs/features/asc1/stateless/sdks.md:267]: @ryanRfox was just making these update and retesting things and I'm not sure this actually works. I tried a basic program with just a simple secret word. 

```
#pragma version 2
arg_0
byte "xxx"
==
```
Call it like this:

```
const args = ["xxx"]
    const lsig = algosdk.makeLogicSig(program, args);
```
And it's always rejected by logic. Am I missing something basic? I can get the int example working but not this.

File: docs/features/asc1/stateless/sdks.md [hunk 2]
```diff
@@ -787,88 +800,49 @@ Delegated Logic Signatures require that the logic signature be signed from a spe
...
+   
+    const tx = await algodClient.sendRawTransaction(rawSignedTxn.blob).do();
     console.log("Transaction : " + tx.txId);    
-    await waitForConfirmation(algodclient, tx.txId);
-})().catch(e => {
-    console.log(e.body.message);
-    console.log(e);
-});
+    await waitForConfirmation(algodClient, tx.txId);
+};
```
[the remark this answers is not in the document]
Comment (edwardgaudio) [reply docs/features/asc1/stateless/sdks.md:835]: Added to the bottom. I was a little hazy on where this should go though.

File: docs/features/asc1/stateless/sdks.md (hunk 1 above)
Comment (edwardgaudio) [reply docs/features/asc1/stateless/sdks.md:266]: I got it to work like this:

```const arg = [...Buffer.from('xxx')];

    const args = [arg];
    const lsig = algosdk.makeLogicSig(program, args);
```
Should we do it like this or recommend something else?

Comment (ryanRfox) [reply docs/features/asc1/stateless/sdks.md:266]: Good catch here as well. Another example for your consideration is located at [application call with arguments](https://developer.algorand.org/docs/features/asc1/stateful/sdks/#call-with-arguments) (which also requires refactoring in a separate PR). It uses `Buffer.from` as we don't yet have an encoder helper for strings. I prefer to include the `appArgs.push` method (even if only adding a single element) to demonstrate that pattern.  

```
// call application with arguments
let ts = new Date(new Date().toUTCString());
console.log(ts)
let appArgs = [];
appArgs.push(new Uint8Array(Buffer.from(ts)));
```

**Review by ryanRfox — APPROVED** [review]
LGTM

**Pull request merged**